nodewatcher/modules/analysis/rogue_nodes/tests.py

```diff
@@ -0,0 +1,28 @@
+import json
+import io
+import os
+
+from django import test as django_test
+
+from nodewatcher.modules.analysis.rogue_nodes.tasks import rogue_node_detection_algorithm
+
+
+class RogueDetectionTest(django_test.TestCase):
+
+    def test_rogue_detection_algorithm(self):
+        print('testing rogue_node_detection_algorithm')
```

Comment:
Oh. Hm. I though you should look into how to correctly do this as test. I think you should simply dynamically create a test suite (Google for that), instead of having one test for all files. So a test suite where you register one test for each file. And then the testing framework should display errors.